appcues/src/main/java/com/appcues/analytics/ExperienceLifecycleEvent.kt

```diff
@@ -85,6 +86,7 @@ internal sealed class ExperienceLifecycleEvent(
             "experienceId" to experience.id.appcuesFormatted(),
             "experienceName" to experience.name,
             "experienceType" to experience.type,
+            "frameId" to experience.renderContext.getFrameId(),
```

Comment:
Is this an empty string for a modal?

Reply:
looks like it is. good catch - ideally a null / trimmed out

Reply:
its null. is that ok?